dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -421,8 +421,8 @@ yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.usage.billInstancesAfter
 yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.usage.defaultSpendingLimit.forUsers "500"
 yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.usage.defaultSpendingLimit.forTeams "0"
 yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.usage.defaultSpendingLimit.minForUsersOnStripe "1000"
-yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.usage.creditsPerMinuteByWorkspaceClass['default'] "0.1666666667"
-yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.usage.creditsPerMinuteByWorkspaceClass['gitpodio-internal-xl'] "0.3333333333"
+yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.workspaceClasses[2].credits.perMinute "0.1666666667"
```

Comment:
It shouldn't be defined here as part of usage, it should be set as part of setting the WS classes elsewhere

Reply:
This was confusing because originally it was `default` and `gitpodio-internal-xl` but the `workspaceClasses` config were `default` and `small`. Fixed!